src/components/Categories.jsx

```diff
@@ -1,22 +1,15 @@
-import React, {useState, useEffect} from 'react'
-
-import ProductCategories from '../utils/mocks/en-us/product-categories.json'
+import React from 'react'
+import PropTypes from 'prop-types'
 
 import {CategoryCard, Grid} from './common'
 
-const Categories = () => {
-
-  const [data, setData] = useState(null)
-
-  useEffect(() => {
-    setData(ProductCategories.results)
-  },[data])
+const Categories = ({data}) => {
 
   return (
     <section className='categories'>
       <Grid col={5} mdCol={2} smCol={1} gap={20}>
         {
-            data?.map((item, index) => 
+            data?.results?.map((item, index) => 
```

Comment:
Keep in mind that tracking elements by array index may be not the best approach, as if by some reason the array mutates, the UI may behave wrong, the best is to use a unique key attached to the item itself when posible, in this case the categorie id would be a nice option. `https://reactjs.org/docs/lists-and-keys.html#keys`